tests/acceptance/disabled-button-test.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+import contains from '../helpers/contains';
+
+    var App, DisabledController;
+
+    module('Acceptance: Disabled AsyncButton', {
```

Comment:
This line and the tests lines have odd indentation, the `module` and `test` lines need to be outdented